### DISCUSSION TITLE
Unable to view repositories

### DISCUSSION THREAD

**Discussion by aminwafi** [community/community#43922] [Mobile]
I tried to navigate to repositories from home page, but it kept showing me 'Something went wrong. Try again'. I also tried signing out and re-login but the issue still persists

![image](https://user-images.githubusercontent.com/41036566/211707699-2e15efc9-dca1-44d9-b565-cc5c614d8371.png)

**Comment by trwstin** [comment]
Do you get the same issue when accessing it from a browser instead of the app? 
Does it happen only when you access your own repos or do you see the same issue on others' repos?

**Reply by aminwafi** [reply]
Hi trwstin, in response to your question, this issue occurs only when I try to access my own repos from the app. I am able to access other repos except my own. Also, I have no issues accessing from the browser. 

Im using an old Iphone and my iOS version is 14.8 by the way, so technically I can't get the latest github version. Currently Im using Github version 1.78. If seems like there is no way to work this around, guess Im getting a new phone :\

**Comment by ghost** [comment]
Hi aminwafi TRT to delete the app and login again

**Reply by aminwafi** [reply]
Hi yuheshpandian, I've already tried your solution but its still not working. It would still download Github version 1.78 which have the same issue :')
